internal/plugin/plugin_test.go

```diff
@@ -19,7 +19,7 @@ import (
 
 func init() {
 	// Set our default log level lower for tests
-	hclog.L().SetLevel(hclog.Trace)
+	hclog.L().SetLevel(hclog.Debug)
```

Comment:
Is this intentional?

Reply:
It was, yes. The trace is just too noisy for general testing.